bin/post_compile

```diff
@@ -80,21 +97,23 @@ heroku-install-cabot-plugins() {
   done
 }
 
+heroku-put-scripts-on-path() {
+  # A better way of doing this perhaps?
+  ln -sf /app/.heroku/vendor/node-4.2.6/bin/* /app/.heroku/python/bin/
```

Comment:
I needed to get the node commands on the path (Django still seems to need them to dynamically compress assets). That being said, perhaps Django was only trying to do this because Debug was set to True.

I can take a look at this tomorrow. It is 6am here :-)